doc/vision.rst

```diff
@@ -49,20 +49,10 @@ From shorter term to longer term:
 - Make the :class:`TableVectorizer` fast, robust, and easy to tune (in
   the sense of hyper-parameter tuning)
 
-- Add a Join-aggregator object, to do feature augmentation on one-to-many
-  correspondences
-
-- Support polars
-
-- Support time series (eg in the aggregations)
-
-- Interpolator join to join across multiple columns without exact
-  correspondences in the keys
-
-- Release (yes we are not planning to release very soon)
+- Better support for time series
```

Comment:
I can't comment on it, but I think the following mention should be removed: "Make the TableVectorizer fast, robust, and easy to tune (in the sense of hyper-parameter tuning)"